docs/core-features/extensions.md

```diff
@@ -78,6 +78,12 @@ sample = [np.array([[3, -1], [2, 4]]), 1]
 assert np.array_equal(circuit.encrypt_run_decrypt(*sample), function(*sample))
 ```
 
+[{% hint style="info" %}
+Multivariate functions can be converted in two ways:
+- **fhe.MultivariateStrategy.PROMOTED**: Inputs (e.g., x, y, ...) of multivariate would be promoted to sum of their bit widths (i.e., `x.bit_width + y.bit_width + ...`) during compilation, so the operation would be a single table lookup but input bit widths would be increased.
+- **fhe.MultivariateStrategy.CASTED**: Inputs (e.g., x, y, ...) of multivariate would be casted to sum of their bit widths (i.e., `x.bit_width + y.bit_width + ...`) during runtime, so the operation would be 3 table lookups but input bit widths would not be affected.
+{% endhint %}]()
```

Comment:
I would try to avoid to add complicated things in this .md, which is used in the "easy" part of the doc

```
## Operations

* [Table Lookups basics](core-features/table_lookups.md)
* [Non-linear operations](core-features/non_linear_operations.md)
* Other operations
  * [Bit extraction](core-features/bit_extraction.md)
  * [Common tips](core-features/workarounds.md)
  * [Extensions](core-features/extensions.md)
```

The hard parts are eg for 

```
* Advanced features
  * [Table Lookups advanced](core-features/table_lookups_advanced.md)
  * [Rounding](core-features/rounding.md)
  * [Truncating](core-features/truncating.md)
  * [Floating points](core-features/floating_points.md)
  * [Comparisons](core-features/comparisons.md)
  * [Min/Max operations](core-features/minmax.md)
  * [Bitwise operations](core-features/bitwise.md)
  * [Direct circuits](compilation/direct_circuits.md)
  * [Tagging](core-features/tagging.md)
```

Reply:
so @umut-sahin could you revert this part please?